src/coreComponents/codingUtilities/tests/testTable.cpp

```diff
@@ -0,0 +1,262 @@
+               "+----------------------------------------------------------------------------------------------------------------------------------------------+--------------+---------------+-----------+-----------+\n\n"
+               );
+  }
+
+  //same but with different values
+  {
+    TableLayout tableLayout( {"Duis fringilla, ligula sed porta fringilla,\nligula wisi commodo felis,ut adipiscing felis dui in enim. Suspendisse malesuada ultrices ante",
+                              "CordX",
+                              "CoordZ",
+                              "Prev\nelement",
+                              "Next\nelement"}, "InternalWellGenerator well_injector1" );
+
+    TableData tableData;
+    tableData.addRow( "value1", "[30.21543]", "3.0", 54, 0 );
+    tableData.addRow( "", "", "", "", "" );
+    tableData.addRow( "value23", "[30.21543]", "30.45465142", 787442, 10 );
+
+    TableTextFormatter tableText( tableLayout );
+    EXPECT_EQ( tableText.toString( tableData ),
+               "\n+---------------------------------------------------------------------------------------------------------------------------------------------------------+\n"
+               "|                                                          InternalWellGenerator well_injector1                                                           |\n"
+               "+--------------------------------------------------------------------------------------------------+--------------+---------------+-----------+-----------+\n"
+               "|                                                     Duis fringilla, ligula sed porta fringilla,  |       CordX  |       CoordZ  |     Prev  |     Next  |\n"
+               "|  ligula wisi commodo felis,ut adipiscing felis dui in enim. Suspendisse malesuada ultrices ante  |              |               |  element  |  element  |\n"
+               "+--------------------------------------------------------------------------------------------------+--------------+---------------+-----------+-----------+\n"
+               "|                                                                                          value1  |  [30.21543]  |          3.0  |       54  |        0  |\n"
+               "|                                                                                                  |              |               |           |           |\n"
+               "|                                                                                         value23  |  [30.21543]  |  30.45465142  |   787442  |       10  |\n"
+               "+--------------------------------------------------------------------------------------------------+--------------+---------------+-----------+-----------+\n\n"
+               );
+  }
+
+  //table with TableLayout::ColumnParam
+  {
+    TableLayout tableLayout( {
```

Comment:
```suggestion
    TableLayout const tableLayout( {
```
Same here, try to add `const` when you can.